dbms/src/Storages/KVStore/MultiRaft/RegionCFDataBase.h

```diff
@@ -52,6 +52,10 @@ struct RegionDataMemDiff
         payload -= other.payload;
         decoded -= other.decoded;
     }
+
+    Type total() const { return payload + decoded; }
+
+    size_t totalSize() const { return payload + decoded; }
```

Comment:
Why we need two method for the same result? Maybe just adding explicit cast when want to assign the result to differnet type?

Reply:
`total()` needs no casting, but is not used very much,
`totalSize()` needs casting, but is widely used in test, so it is we just write it here to eliminate some `static_cast`ing